### ISSUE TITLE
Support webauthn

### ISSUE THREAD

**Issue by urbaniak** [getsentry/sentry#18697]
## Summary

Support superior to u2f standard called webauthn

## Motivation

webauthn is superior to u2f and works in all browsers, even on safari on iOS

**Comment by siedentop** [comment]
Request for clarification: Is this the reason that Sentry does not support Yubikey 5Ci on iOS? 

I tried to use my Yubikey on iPad (6th gen; A1893)  both with Safari and Chrome and it says that the browser does not support a 2FA device.

**Comment by urbaniak** [comment]
yeah, sentry right now supports the old u2f api and on iOS you can only use webauthn

**Comment by BYK** [comment]
I think @dcramer is working on this.

**Comment by dcramer** [comment]
No active work - more complex than my afternoon allowed, but we do need to get it done.

**Comment by maxiuyuan** [comment]
Hello! this is now fixed by #30612 and the prs listed in its description

**Comment by evilscientress** [comment]
I'm running Sentry 22.1.0 self hosted but it is still using the old deprecated U2F api. But looking at this issue, #30612 and the release notes of 21.12.0 the fix should be in the codebase by now. Is there any additional manual step the needs to be made?

I tried adding a new yubikey to an account but even for registering new devices the old U2F api is still used.
It's February so the old api will stop working soon locking people out of their accounts.

I tried it on two separate installation with the same result.

**Comment by BYK** [comment]
@evilscientress I think you need to enable `organizations:webauthn-login` that said it was removed some days ago in #31055.

**Comment by evilscientress** [comment]
Just tried that and enabled the feature in `sentry.conf.py` but no change. Stil uses the old U2F api.

**Comment by BYK** [comment]
@evilscientress just to make sure:

1. You added that string as is to the list here: https://github.com/getsentry/self-hosted/blob/f3027d08ca5e004b481d4b9482004dd0e318dc78/sentry/sentry.conf.example.py#L246
2. And then you restarted your Sentry instances?

**Comment by evilscientress** [comment]
Yep. That's exactly what I did. But I just tested it again and it works now for logging but not for enrolling U2F devices. There the old api is still used.

Also something I noticed is that for some reason on chromium/linux I need to enter the PIN code of my WebAuthn/U2F device but not on chrome/windows. Other WebAuthn implementations don't require the pin code on chromium/linux. (Github, GitLab, etc.)
